distributor-api-v1/operations.md

```diff
@@ -744,14 +744,14 @@ Gives a pricing information for the given configuration.
 | :--- | :--- | :--- | :--- |
 | `OccupancyPrices` | array of [RoomOccupancyAvailability](operations.md#roomoccupancyavailability) | required | Pricing information. |
 
-## Get Payment Gateway   <a id="create-reservation-group"></a>
+## Get Payment Configuration   <a id="get-payment-configuration"></a>
 
-### Request`[PlatformAddress]/api/distributor/v1/payments/getPaymentGateway`   <a id="request-platformaddressapidistributorv1reservationgroupscreate"></a>
+### Request `[PlatformAddress]/api/distributor/v1/hotels/getPaymentConfiguration`   <a id="request-getpaymentconfiguration"></a>
```

Comment:
@dobras 📝 Please create a follow up task where we should fix this formatting for each `Request` (adding a space).

Reply:
https://mews.myjetbrains.com/youtrack/issue/TDD-1726